pre/remap_restart/remap_upper.py

```diff
@@ -387,6 +394,43 @@ def copy_merra2(self):
          bin2nc(dest, ncdest, yaml_file)
        os.remove(dest)
 
+  def link_without_remap(self, restarts_in, suffix):
+     config = self.config
+     in_agrid        = config['input']['shared']['agrid']
+     out_agrid       = config['output']['shared']['agrid']
+     out_levels      = config['output']['air']['nlevel']
+     in_bc_version   = config['input']['shared']['bc_version']
+     out_bc_version  = config['output']['shared']['bc_version']
+     in_stretch      = config['input']['shared']['stretch']
+     out_stretch     = config['output']['shared']['stretch']
+
+     if (in_agrid == out_agrid and \
+         in_bc_version == out_bc_version and in_stretch  == out_stretch) :
```

Comment:
Is the check `in_bc_version==out_bc_version` here included to verify if the topo files are identical?  For two different bcs versions, the topo files may be the same, and we would still remap the upper air restarts without needing to do so.  
(E.g., the topo  files for  NLv3 and v11 bcs are probably the same.)

Reply:
If so, we probably need to verify the topo files instead just compare bc_versions.